app/views/characters/_characters_skills.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="container">
```

Comment:
Same for this file, review your indentation before pushing your files up to Github